backend/frontend/views.py

```diff
@@ -259,15 +261,27 @@ def view_partial_user_board_column(request, user_id, column_id):
     )
 
 
-@user_passes_test(is_super)
 @csrf_exempt
 def action_start_card(request, card_id):
```

Comment:
Don't we need to check is the action is permitted? Like how the DRF does it with perms like:

```python
permission_classes=[
            (
                curriculum_permissions.IsCardAssignee
                & curriculum_permissions.CardCanStart
                & curriculum_permissions.AssigneeHasFinishedReviews
            )
            | (
                HasObjectPermission(
                    permissions=Team.PERMISSION_MANAGE_CARDS,
                    get_objects=_get_teams_from_card,
                )
                & curriculum_permissions.CardCanForceStart
            )
        ],
```

Reply:
I will have a closer at protecting the view